examples/_bootstrap/main.tf

```diff
@@ -0,0 +1,49 @@
+provider "aws" {
+  region = local.region
+}
+
+locals {
+  region   = "eu-west-1"
+  name     = "bootstrap-example"
+  vpc_cidr = "10.0.0.0/16"
+
+  cluster_name = "test-eks-${random_string.suffix.result}"
+}
+
+data "aws_availability_zones" "available" {}
+
+resource "random_string" "suffix" {
+  length  = 8
+  special = false
+}
+
+################################################################################
+# Supporting Resources
+################################################################################
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = local.name
+  cidr = "10.0.0.0/16"
+
+  azs             = data.aws_availability_zones.available.names
+  public_subnets  = [for k, v in data.aws_availability_zones.available.names : cidrsubnet(local.vpc_cidr, 8, k)]
```

Comment:
maybe more clear will be move this to locals?

Reply:
No need because this is used just once.